src/js/configuration_panel/button-listeners.js

```diff
@@ -176,50 +250,75 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                         initialInputData = initialConfiguration[isAxes ? "axes" : "buttons"][inputIndex];
 
                     /**
-                     * Enable the "Save Changes" button if dropdown option is changed.
-                     * Otherwise, disable the button.
+                     * Enable the "Save Changes" and "Discard Changes" buttons if
+                     * dropdown option is changed.
                      */
                     var actionDropdown = configurationMenu.querySelector("select"),
```

Comment:
Apologies for not catching this earlier.  Very soon we should refactor your work to use Fluid ViewComponents rather than direct selectors.  It would make it easier for someone to reuse your work with different markup.  I will write that up for later.

Reply:
Please add a TODO somewhere nearby referencing this ticket:

https://github.com/fluid-lab/gamepad-navigator/issues/40